### PR TITLE
docs: translate API Reference -> `react-dom` -> Client -> `createRoot`

### LINKED ISSUE #454: `createRoot`
Page: https://id.react.dev/reference/react-dom/client/createRoot

Assignee: @chez14

### PR DESCRIPTION
**Pull request by chez14**
Closes #454 

## Description

Translate the `react-dom/client` → `createRoot` page.
Page URL: 
- Original: https://react.dev/reference/react-dom/client/createRoot
- Localized: https://id.react.dev/reference/react-dom/client/createRoot

References:
- #330+
- #454+

### CODE REVIEW

**Comment by chez14** [comment]
Hi @resir014 / @zainfathoni, I've translated the page and this MR is ready for review.

Tough, I still have questions:
- I noticed that some titles such as ["Parameters"][ref-para] and ["Returns"][ref-retr] are not translated in one of the Sepuh's MR, and I honestly feel like this can be translated properly as "Parameter" and "Pengembalian". Should we do that? (In current MR, I left it as is)
- Should I translate following terms?
  - "server-side rendering (SSR)" → "Perenderan pada sisi server (server-side rendering, SSR)"
  - "static site generation (SSG)" → "Pembangunan situs statik (static site generation, SSG)"
  
  Currently I've left it as is and treat it as loan word (term?).

Also note:
- I noticed that loanwords that have been 'glued' with _imbuhan_ such as ["merender"][ref-render] currently don't need to be italicized or separated with `-`, I applied this rule for "render" and "unmount".
- I did not translate the error messages in the Troubleshooting part because I don't think error messages in React is localized too. Please let me know if this should actually be translated too.

Thank you so much!

[ref-para]: https://github.com/reactjs/id.react.dev/pull/351/files#diff-434456b83005426d91d6683a04db292dea22bc6fee2608038715c6b807f28882L35
[ref-retr]: https://github.com/reactjs/id.react.dev/pull/351/files#diff-434456b83005426d91d6683a04db292dea22bc6fee2608038715c6b807f28882L39
[ref-render]: https://github.com/reactjs/id.react.dev/pull/351/files#r1179192132

**Comment by zainfathoni** [comment]
> Hi @resir014 / @zainfathoni, I've translated the page and this MR is ready for review.
> 
> Tough, I still have questions:
> 
> * I noticed that some titles such as ["Parameters"](https://github.com/reactjs/id.react.dev/pull/351/files#diff-434456b83005426d91d6683a04db292dea22bc6fee2608038715c6b807f28882L35) and ["Returns"](https://github.com/reactjs/id.react.dev/pull/351/files#diff-434456b83005426d91d6683a04db292dea22bc6fee2608038715c6b807f28882L39) are not translated in one of the Sepuh's MR, and I honestly feel like this can be translated properly as "Parameter" and "Pengembalian". Should we do that? (In current MR, I left it as is)

Yes, as discussed over here, https://github.com/reactjs/id.react.dev/discussions/337#discussioncomment-5769320, we agreed to translate them.

> * Should I translate following terms?
>   
>   * "server-side rendering (SSR)" → "Perenderan pada sisi server (server-side rendering, SSR)"
>   * "static site generation (SSG)" → "Pembangunan situs statik (static site generation, SSG)"
>   
>   Currently I've left it as is and treat it as loan word (term?).

Those are very popular phrases recently in the frontend community. I think translating them prematurely might lead to more confusions. Therefore, let's keep them as loan words, but don't forget to italicize them.

> Also note:
> 
> * I noticed that loanwords that have been 'glued' with _imbuhan_ such as ["merender"](https://github.com/reactjs/id.react.dev/pull/351/files#r1179192132) currently don't need to be italicized or separated with `-`, I applied this rule for "render", "unmount", "hydrate", "log".

I disagree. In [PUEBI](https://ivanlanin.github.io/puebi/tanda-baca/tanda-hubung/), we have to italicize them while connecting the prefix and suffix with `-`.
image

> * I did not translate the error messages in the Troubleshooting part because I don't think error messages in React is localized too. Please let me know if this should actually be translated too.

Yes, please leave the error messages in the Troubleshooting part because that's what the readers will encounter while actually running the code and using React in the real world.

> Thank you so much!

Thank you for your comprehensive considerations and contribution!

**Review by r17x — CHANGES_REQUESTED** [review]

**Comment by r17x** [comment]
Hello @chez14, Do you need a help for this PRs?

Thanks

**Comment by chez14** [comment]
Hi @r17x,

Ah thank you for reaching back, I'm waiting for more feedback so I can edit them in one go.

Should I just patch this, or... wait for other reviewer first?

Thanks!

**Comment by resir014** [comment]
@chez14 You can start working on any suggested changes now. We can come back and review it later.

**Comment by chez14** [comment]
Hi @r17x, @resir014, and @zainfathoni,

I have updated the PR as requested. 

Please let me know if you have any feedback or questions. Thank you!

File: src/content/reference/react-dom/client/createRoot.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: createRoot
 
 <Intro>
 
-`createRoot` lets you create a root to display React components inside a browser DOM node.
+`createRoot` memungkinkan Anda untuk membuat sebuah akar untuk menampikan komponen React dalam node DOM pada browser.
```
Comment (r17x) [inline src/content/reference/react-dom/client/createRoot.md:7]: Peramban

File: src/content/reference/react-dom/client/createRoot.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ const root = createRoot(domNode, options?)
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `createRoot(domNode, options?)` {/*createroot*/}
 
-Call `createRoot` to create a React root for displaying content inside a browser DOM element.
+Panggil `createRoot` untuk membuat akar React untuk menampilkan konten di elemen DOM di browser.
```
Comment (r17x) [inline src/content/reference/react-dom/client/createRoot.md:23]: Idem https://github.com/reactjs/id.react.dev/pull/514/files#r1248843137

**Review by r17x — APPROVED** [review]

**Comment by r17x** [comment]
@zainfathoni @resir014 This PRs have been updated yet, need re-review.

**Comment by r17x** [comment]
@resir014 could you review this PR

**Review by resir014 — APPROVED** [review]

**Pull request merged**